prometheus-exporters/vrops-exporter/alerts/datastore.alerts

```diff
@@ -173,6 +173,22 @@ groups:
       description: "{{ $labels.type }} datastore {{ $labels.datastore }} utilization > 90%. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
       summary: "{{ $labels.type }} datastore {{ $labels.datastore }} utilization > 90%. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
 
+  - alert: NVMeDataStoreCapacityWarning
+    expr: >
+      max_over_time(vrops_datastore_diskspace_total_usage_gigabytes{datastore=~".+swap"}[10m]) / max_over_time(vrops_datastore_diskspace_capacity_gigabytes[10m]) >=  0.85
```

Comment:
`max_over_time` should not be needed anymore, should it?

Reply:
The only gab I found today was because of a hickup in a single collector.

Reply:
remove it completely then